packages/pn-commons/src/components/CustomTooltip.tsx

```diff
@@ -7,12 +8,11 @@ import Tooltip from '@mui/material/Tooltip';
 type Props = {
   tooltipContent: any;
   openOnClick: boolean;
```

Comment:
restore children props. In react18 this property becomes mandatory

Do it for ALL components that use children